website_customer_also_purchased/static/src/less/fonts/slick.svg

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
Please leave it like this, I just saw the less file down here and means a lot of changes to the less file